File: W2/Request4.py
def main():
    def twoSum(nums, target):
        for i in range(len(nums)):
            a = nums[i]
            b = target - a
            for j in range(i+1, len(nums)):
               if b == nums[j]:
                  return (i, j)
    result=twoSum([2, 11, 7, 15], 9)
    print(result)


if __name__ == '__main__':
    main()

File: W2/request4.js
 function twoSum(nums, target) {
            for (var i = 0; i < nums.length; i++) {
                for (var j = 0; j < nums.length; j++) {
                    if (nums[i] + nums[j] == target) {
                        return ('[' + i + ',' + j + ']')
                    };
                };
            }
        }
        result = twoSum([2, 11, 7, 15], 9)
        console.log(result)
